base/system/diskpart/lang/pt-PT.rc

```diff
@@ -0,0 +1,392 @@
+    IDS_STATUS_YES "Sim"
+    IDS_STATUS_NO "Não"
+    IDS_STATUS_DISK_HEALTHY "Saudável"
+    IDS_STATUS_DISK_SICK "Ruim"
+    IDS_STATUS_UNAVAILABLE "Indisponivél"
+    IDS_STATUS_ONLINE "Ativo"
+    IDS_STATUS_OFFLINE "Inativo"
+    IDS_STATUS_NO_MEDIA "sem disco"
+END
+
+/* CMD Messages for commands */
+STRINGTABLE
+BEGIN
+    IDS_MSG_ARG_SYNTAX_ERROR "Os argumentos especeficados para esse comando não são válidos.\nPara mais informção sobre esse comando:"
+END
+
+/* Help Command Descriptions */
+STRINGTABLE
+BEGIN
+    IDS_HELP_CMD_DESC_ACTIVE "ACTIVE\t\t- Marca a partição selecionada como ativa.\n"
+    IDS_HELP_CMD_DESC_ADD "ADD\t\t-  Duplica o volume.\n"
+    IDS_HELP_CMD_DESC_ASSIGN "ASSIGN\t\t- Define a letra da partição selecionada.\n"
+    IDS_HELP_CMD_DESC_ATTACH "ATTACH\t\t- Anexa um arquivo de disco virtual.\n"
+    IDS_HELP_CMD_DESC_ATTRIBUTES "ATTRIBUTES\t- Manipula o volume ou os atributo do disco.\n"
+    IDS_HELP_CMD_DESC_AUTOMOUNT "AUTOMOUNT\t- Ativa ou desativa a montagem automática de discos.\n"
+    IDS_HELP_CMD_DESC_BREAK "BREAK\t\t- Quebrar duplição.\n"
+    IDS_HELP_CMD_DESC_CLEAN "CLEAN\t\t- Apagar a informção de configuração, or toda a informção, desliga\n\t\t  o disco.\n"
+    IDS_HELP_CMD_DESC_COMPACT "COMPACT\t\t- Tenta reduzir o tamanho físico do ficheiro.\n"
+    IDS_HELP_CMD_DESC_CONVERT "CONVERT\t\t- Converter entre diferentes formatos.\n"
+    IDS_HELP_CMD_DESC_CREATE "CREATE\t\t- Criar volume,partição ou disco virtual.\n"
+    IDS_HELP_CMD_DESC_DELETE "DELETE\t\t- Apaga um objeto.\n"
+    IDS_HELP_CMD_DESC_DETACH "DETACH\t\t- Ejeta um disco virtual.\n"
+    IDS_HELP_CMD_DESC_DETAIL "DETAIL\t\t- Fornece detalhes sobre um objeto.\n"
+    IDS_HELP_CMD_DESC_EXIT "EXIT\t\t- Sair do particionamento de disco.\n"
+    IDS_HELP_CMD_DESC_EXPAND "EXPAND\t\t- Aumentar o tamanho do disco virtual.\n"
+    IDS_HELP_CMD_DESC_EXTEND "EXTEND\t\t- Estender um volume.\n"
+    IDS_HELP_CMD_DESC_FS "FILESYSTEMS\t- Mostra sistemas de arquivos atuais e suportados no volume.\n"
+    IDS_HELP_CMD_DESC_FORMAT "FORMAT\t\t- Formatar volume ou partição.\n"
+    IDS_HELP_CMD_DESC_GPT "GPT\t\t- Definir atributos da partição GPT atualmente selecionada.\n"
+    IDS_HELP_CMD_DESC_HELP "HELP\t\t- Mostra a lista de comandos.\n"
+    IDS_HELP_CMD_DESC_IMPORT "IMPORT\t\t- Importar grupo de discos.\n"
+    IDS_HELP_CMD_DESC_INACTIVE "INACTIVE\t- Maracr a partição selecionada como inativa.\n"
+    IDS_HELP_CMD_DESC_LIST "LIST\t\t- Mostra lista de objetos.\n"
+    IDS_HELP_CMD_DESC_MERGE "MERGE\t\t- Mescla um disco filho com seus pais.\n"
+    IDS_HELP_CMD_DESC_OFFLINE "OFFLINE\t\t- Marca como offline um objeto que está atualmente online.\n"
+    IDS_HELP_CMD_DESC_ONLINE "ONLINE\t\t- Marca como online um objeto que está atualmente offline.\n"
+    IDS_HELP_CMD_DESC_RECOVER "RECOVER\t\t- Atualiza o estado de todos os discos no pacote inválido,\n\t\t  and resynchronizes mirrored volumes and RAID5 volumes\n\t\t  that have stale plex or parity data.\n"
```

Comment:
The string is not fully translated

Reply:
não entendi

Reply:
já vi

Reply:
como vou eu traduzir plex?

Reply:
Actualiza o estado de todos os discos no pacote inválido,\n\t\t e ressincroniza volumes espelhados e volumes RAID5 \n\t\t que possuem encadeamento obsoleto ou dados de paridade\n\t.

Reply:
ou que possuam plexo obsoleto ou dados de paridade\n\t.

Reply:
ok